src/kontrol/cli.py

```diff
@@ -52,3 +53,9 @@ def k_gen_args(self) -> ArgumentParser:
             help='Extra modules to import into generated main module.',
         )
         return args
+
+    @cached_property
+    def kompile_target_args(self) -> ArgumentParser:
+        args = ArgumentParser(add_help=False)
+        args.add_argument('--target', type=KompileTarget, help='[haskell-booster|maude]')
```

Comment:
I suppose this flag could be used to pass other targets by mistake. You could restrict them using `choices=[KompileTarget.HASKELL_BOOSTER, KompileTarget.MAUDE],`.

Reply:
I believe I tried that and it broke, but I'll try again and see if I am misremembering.